src/cli/impl/Validators.cpp

```diff
@@ -107,7 +110,7 @@ std::optional<bool> Validate::Confirm(const std::string& symbol)
 }
 std::optional<std::string> Validate::FileName(const std::string& file_name)
 {
-    const std::string pattern = R"(\w{0,}.\w{0,})";
+    const std::string pattern = R"(\w{0,}\.\w{0,})";
```

Comment:
Not a part of this review, but in general, avoid regexp, it is notoriously slow.
@IlliaPlaksa @AntonApraksin @KharchenkoMaks @antonovchtd @MichaelKupriianov